up42/utils.py

```diff
@@ -300,6 +320,116 @@ def _plot_images(
     plt.show()
 
 
+def _map_images(
+    plot_file_format: List[str],
+    result_df: GeoDataFrame,
+    filepaths,
+    aoi=None,
+    show_images=True,
+    show_features=False,
+    name_column: str = "id",
+    save_html: Path = None,
+):
```

Comment:
can you add a return type? I guess will be folium.Map?

Reply:
Nice, I did not think about it, I will add it then.